addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -618,6 +618,9 @@ export class Wysiwyg extends Component {
 
         if (options.snippets) {
             $(this.odooEditor.document.body).addClass('editor_enable');
+            if (localization.direction === "rtl") {
+                this.odooEditor.document.body.setAttribute("is-rtl-backend", "true");
+            }
```

Comment:
Isn't there already an `o_rtl` class or attribute that you can use instead of adding a new one (since you did "closest" that elsewhere) ? :slightly_smiling_face: